app/Repositories/Order/OrderRepositoryInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Repositories\Order;
+
+use Illuminate\Http\Request;
+use App\Repositories\RepositoryInterface;
+
+interface OrderRepositoryInterface extends RepositoryInterface
+{
+    //view orderDeatils
+    public function viewDetails($id);
+
```

Comment:
```suggestion
    **
     * Get detail (--> Description of this function)
     * @param  int $id
     * @return \App\Eloquent\Detail
     */
    public function viewDetails($id);

```
viết comment kiểu ntn nhé em

Reply:
vâng chị